src/OrchardCore.Modules/OrchardCore.ContentFields/Settings/ContentPickerFieldSettings.cs

```diff
@@ -10,5 +11,10 @@ public class ContentPickerFieldSettings
         public bool DisplayAllContentTypes { get; set; }
         public string[] DisplayedContentTypes { get; set; } = Array.Empty<string>();
         public string[] DisplayedStereotypes { get; set; } = Array.Empty<string>();
+
+        /// <summary>
+        /// The pattern used to build custom title
+        /// </summary>
+        public string TitlePattern { get; set; } = "{{ Model.ContentItem.DisplayText }}";
```

Comment:
add attribute [DefaultValue("{{ Model.ContentItem.DisplayText }}")] to the new property since it has a custom default value.